### PR TITLE
Crypto Market: Capstone Project Submission

### PR DESCRIPTION
**Pull request by bruk19**
In this PR I have implemented the capstone project requirements. I have built a web app on top of React and Redux that fetches data from the Crypto database API.

In this PR I have:

-  Installed React
- Installed and configured linters
- Installed and configured Redux
- Installed and configured React Router
- Added the required routes: Home and per-category unique route
- Added the filter functionality on the home page
-  Implemented the redux actions and reducers to fetch and maintain the global store data
- Installed and customized tailwind to follow the design guidelines (color, fonts and structure)
- Added tests for the entire application
- Deployed the application to Netlify
- Fixed linter errors

#VIDEO:  https://loom.com/share/ca205db5c6ff4ffe96f2ceb6de5a962e

### CODE REVIEW

File: src/_tests_/Pages.test.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import Pages from '../Components/CryptosPage';
+
+it('Test if The CryptoPage render correctly', () => {
+  const TREE = TestRenderer.create(
+    <Provider store={store}>
+      <BrowserRouter>
+        <Pages />
+      </BrowserRouter>
+    </Provider>,
+  );
```
Comment (dasileker) [inline src/_tests_/Pages.test.js]: - [ ] Please correct this test suits it's failing.

File: src/_tests_/Nav.test.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import Nav from '../Components/Nav';
+
+it('Test if header render correctly', () => {
+  const TREE = TestRenderer.create(
+    <Provider store={store}>
+      <BrowserRouter>
+        <Nav />
+      </BrowserRouter>
+    </Provider>,
+  );
```
Comment (dasileker) [inline src/_tests_/Nav.test.js]: - [ ] Also this test suit is falling please correct it.

**Review by dasileker — COMMENTED** [review]
### **Hi @bruk19,** 

### **Good job so far!**
There are some issues that you still need to work on to go to the next project but you are almost there!

## **Highlights**
- Linters are green ✅.
- Commit are consistent 👌.
- The app runs smoothly 👌.

## Required Changes ♻️
_Check the comments under the review._

- **Nice work on the capstone however the test suits are falling due to wrong pieces added, you can also add a jest library to help you better to correct this issue.**
- **Also please make sure you deploy the app using the platform provided by microverse( netlify ), and also please add the repo link to your Getting started in your readme file.**

Capture d’écran 2022-08-04 141327

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @bruk19 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**